tardis/montecarlo/base.py

```diff
@@ -272,16 +272,16 @@ def calculate_radiationfield_properties(self):
         Parameters
         ----------
 
-        nubar_estimator : ~np.ndarray (float)
+        nubar_estimator : np.ndarray (float)
```

Comment:
I [found out](https://www.jetbrains.com/help/pycharm/2016.3/type-hinting-in-pycharm.html#legacy) that PyCharm suggests `np.ndarray[float]`. Check the link for more info. Would be nice to also fix any other instances that match the table in the link